src/actions/WalletActions.tsx

```diff
@@ -21,6 +26,104 @@ export interface SelectWalletTokenParams {
+      const { currencyCode, displayName } = getToken(wallet, tokenId) ?? {}
+      return `${displayName} (${currencyCode})`
+    })
+    const tileTitle = tokenIds.length > 1 ? s.strings.activate_wallet_tokens_scene_tile_title : s.strings.activate_wallet_token_scene_tile_title
+    const tileBody = tokensText.join(', ')
+
+    const { networkFee } = activationQuote
+    const { nativeAmount: nativeFee, currencyPluginId, tokenId: feeTokenId } = networkFee
+    if (currencyPluginId !== pluginId) throw new Error('Internal Error: Fee asset mismatch.')
+
+    const paymentCurrencyCode = getCurrencyCode(wallet, feeTokenId)
+
+    const exchangeNetworkFee = await wallet.nativeToDenomination(nativeFee, paymentCurrencyCode)
+    const feeDenom = getDisplayDenomination(state, pluginId, paymentCurrencyCode)
+    const displayFee = div(nativeFee, feeDenom.multiplier, log10(feeDenom.multiplier))
+    let fiatFee = convertCurrencyFromExchangeRates(state.exchangeRates, paymentCurrencyCode, fiatCurrencyCode, exchangeNetworkFee)
+    if (lt(fiatFee, '0.001')) fiatFee = '<0.001'
+    fiatFee = round(fiatFee, -3)
+    const feeString = `${displayFee} ${feeDenom.name} (${fiatFee} ${fiatCurrencyCode.replace('iso:', '')})`
+
+    navigation.navigate('confirmScene', {
+      titleText: s.strings.activate_wallet_token_scene_title,
+      bodyText: s.strings.activate_wallet_token_scene_body,
+      infoTiles: [
+        { label: tileTitle, value: tileBody },
+        { label: s.strings.mining_fee, value: feeString }
+      ],
+      onConfirm: (resetSlider: () => void) => {
+        if (lt(wallet.balances[paymentCurrencyCode] ?? '0', nativeFee)) {
+          const msg = tokenIds.length > 1 ? s.strings.activate_wallet_tokens_insufficient_funds_s : s.strings.activate_wallet_token_insufficient_funds_s
+          Airship.show<'ok' | undefined>(bridge => (
+            <ButtonsModal
+              bridge={bridge}
+              title={s.strings.create_wallet_account_unfinished_activation_title}
+              message={sprintf(msg, feeString)}
+              buttons={{ ok: { label: s.strings.string_ok } }}
+            />
+          ))
+          navigation.pop()
+          return
+        }
+
+        const name = activateWalletName[pluginId]?.name ?? s.strings.activate_wallet_token_transaction_name_category_generic
+        const notes = activateWalletName[pluginId]?.notes ?? s.strings.activate_wallet_token_transaction_notes_generic
+        activationQuote
+          .approve({
+            metadata: {
+              name,
+              category: `Expense:${s.strings.activate_wallet_token_transaction_name_category_generic}`,
+              notes
+            }
+          })
+          .then(result => {
+            showToast(s.strings.activate_wallet_token_success, ACTIVATION_TOAST_AUTO_HIDE_MS)
+            navigation.pop()
+          })
+          .catch(e => {
+            navigation.pop()
+            console.error(e)
+            showError(e)
```

Comment:
`showError` already includes the `console.log`, so don't do that.

Reply:
will fix